keras_rs/src/utils/doc_string_utils.py

```diff
@@ -0,0 +1,48 @@
+import re
+import textwrap
+from typing import Any
+
+
+def format_docstring(template: str, width: int = 80, **kwargs: Any) -> str:
```

Comment:
Why is this needed?

Reply:
The doc-string has "in-line" variables like "concept_sentence". When I do a .format(), the line goes overboard. When I have a newline in concept_sentence, the next line gets messed up